[Bug]: valgrind build failed

### Problem description

[valgrind.txt](https://github.com/termux/termux-packages/files/9685365/valgrind.txt)


### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

ld: error: duplicate symbol: __aeabi_memclr
>>> defined at aeabi_memset.S.o:(__aeabi_memclr) in archive /home/builder/.termux-build/_cache/android-r25b-api-24-v0/lib64/clang/14.0.6/lib/linux/libclang_rt.builtins-arm-android.a
>>> defined at m_main.c:2645 (/home/builder/.termux-build/valgrind/src/coregrind/m_main.c:2645)
>>>            libnolto_coregrind_arm_linux_a-m_main.o:(.text+0x59E4) in archive ../coregrind/libcoregrind-arm-linux.a

ld: error: duplicate symbol: __aeabi_memclr4
>>> defined at aeabi_memset.S.o:(__aeabi_memclr) in archive /home/builder/.termux-build/_cache/android-r25b-api-24-v0/lib64/clang/14.0.6/lib/linux/libclang_rt.builtins-arm-android.a
>>> defined at m_main.c:2651 (/home/builder/.termux-build/valgrind/src/coregrind/m_main.c:2651)
>>>            libnolto_coregrind_arm_linux_a-m_main.o:(.text+0x59F0) in archive ../coregrind/libcoregrind-arm-linux.a

ld: error: duplicate symbol: __aeabi_memclr8
>>> defined at aeabi_memset.S.o:(__aeabi_memclr) in archive /home/builder/.termux-build/_cache/android-r25b-api-24-v0/lib64/clang/14.0.6/lib/linux/libclang_rt.builtins-arm-android.a
>>> defined at m_main.c:2657 (/home/builder/.termux-build/valgrind/src/coregrind/m_main.c:2657)
>>>            libnolto_coregrind_arm_linux_a-m_main.o:(.text+0x59FC) in archive ../coregrind/libcoregrind-arm-linux.a
clang-14: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [Makefile:1114: memcheck-arm-linux] Error 1
make[3]: Leaving directory '/home/builder/.termux-build/valgrind/build/memcheck'
make[2]: *** [Makefile:1425: all-recursive] Error 1
make[2]: Leaving directory '/home/builder/.termux-build/valgrind/build/memcheck'
make[1]: *** [Makefile:898: all-recursive] Error 1
make[1]: Leaving directory '/home/builder/.termux-build/valgrind/build'

--- a/packages/valgrind/android-memcpy.patch
+++ b/packages/valgrind/android-memcpy.patch
@@ -9,31 +9,14 @@ When we link against clang's builtin library
 
 Hopefully it is fine to remove these overrides now.
 
---- ./coregrind/m_main.c.orig	2022-01-02 12:15:07.892429627 +0100
-+++ ./coregrind/m_main.c	2022-01-02 12:24:20.961799164 +0100
-@@ -2595,25 +2595,6 @@
-         || defined(VGPV_mips32_linux_android) \
-         || defined(VGPV_arm64_linux_android))
+--- a/coregrind/m_main.c
++++ b/coregrind/m_main.c
+@@ -2633,7 +2633,7 @@
+ #endif /* defined(VGP_arm_linux) */
  
--/* Replace __aeabi_memcpy* functions with vgPlain_memcpy. */
--void *__aeabi_memcpy(void *dest, const void *src, SizeT n);
--void *__aeabi_memcpy(void *dest, const void *src, SizeT n)
--{
--    return VG_(memcpy)(dest, src, n);
--}
--
--void *__aeabi_memcpy4(void *dest, const void *src, SizeT n);
--void *__aeabi_memcpy4(void *dest, const void *src, SizeT n)
--{
--    return VG_(memcpy)(dest, src, n);
--}
--
--void *__aeabi_memcpy8(void *dest, const void *src, SizeT n);
--void *__aeabi_memcpy8(void *dest, const void *src, SizeT n)
--{
--    return VG_(memcpy)(dest, src, n);
--}
--
- /* Replace __aeabi_memclr* functions with vgPlain_memset. */
- void *__aeabi_memclr(void *dest, SizeT n);
- void *__aeabi_memclr(void *dest, SizeT n)
+ /* Some Android helpers.  See bug 368529. */
+-#if defined(__clang__) \
++#if 0 && defined(__clang__) \
+     && (defined(VGPV_arm_linux_android) \
+         || defined(VGPV_x86_linux_android) \
+         || defined(VGPV_mips32_linux_android) \

